libdrgn/orc_info.c

```diff
@@ -318,7 +315,9 @@ static struct drgn_error *drgn_read_orc_sections(struct drgn_module *module)
 		err = read_elf_section(orc_header_scn, &orc_header);
 		if (err)
 			return err;
-		module->orc.version = orc_version_from_header(orc_header);
+		module->orc.version = -1;
+		if (orc_header->d_size == 20)
```

Comment:
By the end of this PR, 20 is hard-coded in a bunch of places. Let's define a constant for it.